GVRf/Framework/src/org/gearvrf/GVRVertexBoneData.java

```diff
@@ -0,0 +1,33 @@
+package org.gearvrf;
+
+public final class GVRVertexBoneData extends GVRComponent implements PrettyPrint {
+    /**
+     * Constructor.
+     */
+    public GVRVertexBoneData(GVRContext gvrContext, GVRMesh mesh) {
+        super(gvrContext, NativeVertexBoneData.get(mesh.getNative()), false /* not owned */);
```

Comment:
this instance of VertexBoneData is a part of the native mesh class -- not a pointer. it does not need to be freed separately.